CHANGELOG.md

```
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.1.44-beta.1](https://github.com/bandohq/widget/compare/v0.1.44-beta.0...v0.1.44-beta.1) (2025-08-12)
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)